requirements.in

```diff
@@ -7,6 +7,7 @@ itsdangerous==0.24
 pbr==1.10.0
 protobuf==3.7.0
 pyopenssl==18.0.0
+PyYAML==5.2 # Support for Python 3.4
```

Comment:
Support for Python 3.4 working with what - coverage reporting?

Reply:
No this is something that was raised as I was testing.  When I did a fresh install of the SDK, PyYAML was using a new version (5.3 I think) which specified it was not compatible with Python 3.4.  I've pinned it here to 5.2 which is compatible with Python 3.4.

Reply:
Ok, could we be slightly more descriptive then - something like `5.3+ does not support Python 3.4`?

Reply:
Yep 👍 I'll change that now

Reply:
Fixed in 113f774